frontend/src/scenes/settings/environment/proxyLogic.ts

```diff
@@ -68,7 +69,10 @@ export const proxyLogic = kea<proxyLogicType>([
     listeners(({ actions, values }) => ({
         collapseForm: () => actions.loadRecords(),
         deleteRecordFailure: () => actions.loadRecords(),
-        createRecordSuccess: () => actions.loadRecords(),
+        createRecordSuccess: () => {
+            actions.loadRecords()
+            posthog.capture('proxy_record_created')
```

Comment:
Can you update the event name to follow the lowercase sentence case pattern we use? We should be more specific - maybe something like `managed proxy record created`.

I forgot to mention this but Raquel has been working on moving these events to the backend so ad blockers won't block them. Any crud operations where the events happen on the backup are good candidates for that. Do you want to take a stab at doing this on on the backend?

Reply:
yes for sure! I'm assuming that we can just abandon this frontend change and add the log solely in the backend. Let me know if I'm missing anything

Reply:
Yep, that's probably easiest. Apologies for missing that context.